test-support/helpers/ember-power-select.js

```diff
@@ -61,6 +61,23 @@ export function clickTrigger(scope, options = {}) {
   nativeMouseDown(selector, options);
 }
 
+export function nativeTouchDown (selectorOrDomElement) {
+  let event = new window.Event('touchdown', { bubbles: true, cancelable: true, view: window });
+  let target;
+
+  if (typeof selectorOrDomElement === 'string') {
+    target = Ember.$(selectorOrDomElement)[0];
+  } else {
+    target = selectorOrDomElement;
+  }
+  Ember.run(() => target.dispatchEvent(event));
+}
+
+export function touchTrigger () {
+  let selector = '.ember-power-select-trigger';
```

Comment:
One question. A thought "tap" is compound of a `touchstart` followed by `touchend` in which there wasn't any `touchmove` in between (otherwise is a drag).
I never heard of this `touchdown`, how does it work if I'm adding events for `touch{start|end}`?